stix_shifter_modules/msatp/stix_translation/query_constructor.py

```diff
@@ -269,6 +232,52 @@ def _lookup_comparison_operator(self, expression_operator):
                 "Comparison operator {} unsupported for MSATP connector".format(expression_operator.name))
         return self.comparator_lookup.get(expression_operator)
 
+    # get atomicQuery, map the query to tables
+    def __eval_comparison_exp_map(self, expression):
+        stix_object, stix_field = expression.object_path.split(':')
+        # add later for mac
+        # value = expression.value.values if hasattr(expression.value, 'values') else expression.value
+        # value_type = self._check_value_type(value, expression)
```

Comment:
wondering if the comments still needs to be there?

Reply:
They are not needed, I removed them